ddtrace/contrib/aioredis/patch.py

```diff
@@ -12,13 +12,17 @@
 from ...constants import ANALYTICS_SAMPLE_RATE_KEY
 from ...constants import SPAN_MEASURED_KEY
 from ...ext import SpanTypes
+from ...ext import db
 from ...ext import net
 from ...ext import redis as redisx
 from ...internal.utils.formats import stringify_cache_args
 from ..redis.util import _trace_redis_cmd
 from ..redis.util import _trace_redis_execute_pipeline
 
 
+DBMS_NAME = "redis"
```

Comment:
We can move this constant to ddtrace/ext/redis.py or just use `ddtrace.ext.redis.APP`  (I don't think this constant is even used anymore but it has the value "redis")

Reply:
I completely missed that this module was there, I’ll use that value.